Clinical-Genomics/chanjo-report
# Dont put geneids on session object

### This PR adds | fixes:
- (Scout) session issues with large gene lists

**How to prepare for test**:
- [ ] `ssh` to ...
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
- add genes and see that both report and pdf conversion still work
- add many genes and note that you are no longer kicked out of your session

### Expected outcome:

### Review:

This [version](https://semver.org/) is a:
